rir/R/rir.R

```diff
@@ -6,14 +6,16 @@ rir.markFunction <- function(what, which,
                              DisableInline=NA,
                              DisableAllSpecialization=NA,
                              DisableArgumentTypeSpecialization=NA,
-                             DisableNumArgumentsSepcialization=NA) {
+                             DisableNumArgumentsSepcialization=NA,
+                             Annotated=NA) {
     doIt = function(n) {
         .Call("rirMarkFunction", what, n,
               Reopt,
               ForceInline, DisableInline,
               DisableAllSpecialization,
               DisableArgumentTypeSpecialization,
-              DisableNumArgumentsSepcialization);
+              DisableNumArgumentsSepcialization,
+              Annotated);
```

Comment:
Can we rename this to use something meaningful? DepromisedArgs, e.g.?

Reply:
sure